package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "@empathyco/x-adapter": "^8.0.0-alpha.6",
     "@empathyco/x-adapter-platform": "^1.0.0-alpha.32",
-    "@empathyco/x-components": "3.0.0-alpha.162",
+    "@empathyco/x-components": "3.0.0-alpha.169",
```

Comment:
Yep, I have just updated all our internal dependencies, although I didn't update some packages to the last version because the archetype has to be updated in other tasks.
Feel free to check if I missed something 👌